src/manage-apps/manage-repos.md

```diff
@@ -14,21 +14,21 @@ A repository is developer-maintained online resource that hosts applications' pa
 
 Pop!\_OS includes these built-in repositories:
 
-- **http://apt.pop-os.org/ubuntu/** - This is System76's Ubuntu mirror, which is available globally via System76's CDN.
+- **http://apt.pop-os.org/ubuntu/** - This is System76's Ubuntu mirror, which is available via a global CDN.
 - **https://apt.pop-os.org/release** - This repository provides the released versions of Pop!_OS packages.
 - **https://apt.pop-os.org/proprietary** - This repository contains a number of applications not packaged by Ubuntu, or that are out of date on Ubuntu.
 
-[See this page](https://apt-origin.pop-os.org/) to add these apt repositories.
+[See this page](https://apt-origin.pop-os.org/) to add these default apt repositories if they are somehow removed.
```

Comment:
`if they are somehow removed` is not good wording for an official document that's supposed to be authoritative. "Somehow" in particular is what I have a problem with. It makes it sound like the document doesn't know what's going on.

Unless I am specifically suggesting wording in a review comment, you do not need to pull the specific words that I use to explain concepts to you into the article. Word things so they make the most sense to readers.

Reply:
Fixed this with cde095b.